lib/u3d/commands.rb

```diff
@@ -65,17 +65,15 @@ def list_available(options: {})
         ver = options[:unity_version]
         os = valid_os_or_current(options[:operating_system])
         rl = options[:release_level]
-        cache = Cache.new(force_os: os, force_refresh: options[:force])
 
-        return UI.error "Version #{ver} is not in cache" if ver && cache[os.id2name]['versions'][ver].nil?
+        cache_versions = cache_versions(os, force_refresh: options[:force])
 
-        versions = if ver
-                     { ver => cache[os.id2name]['versions'][ver] }
-                   else
-                     cache[os.id2name]['versions']
-                   end
+        if ver
+          return UI.error "Version #{ver} is not in cache" if cache_versions[ver].nil?
```

Comment:
Maybe we could use `UI.user_error!` directly, which makes more sense imo.

Reply:
I thought about that, as that's we do in the other lanes. I will consider this change in a separate commit as it would be a functional change.